core/src/main/java/org/infinispan/topology/CacheTopologyControlCommand.java

```diff
@@ -36,8 +36,8 @@
       JOIN,
       // A member is signaling that it wants to leave the cluster.
       LEAVE,
-      // A member is confirming that it finished the rebalance operation.
-      REBALANCE_CONFIRM,
+      // A member is confirming that it has finished the topology change during rebalance.
+      REBALANCE_PHASE_CONFIRM,
```

Comment:
I think the comment  describes exactly that: finishing topology change = change from topology with one id to second id. I could be more verbose, saying that this doesn't occur when the target topology is STABLE one.